tensorflow/contrib/makefile/Makefile

```diff
@@ -910,12 +910,18 @@ $(PROTO_TEXT_OBJS) : $(PROTO_TEXT_PB_H_FILES)
 
 # Ensures we link CoreFoundation as it is used for time library when building
 # for Mac.
+ifeq ($(TARGET),OSX)
+  HOST_LDOPTS += -framework CoreFoundation
+  LIBS += -framework CoreFoundation
+endif
 ifeq ($(TARGET),IOS)
   ifeq ($(IOS_ARCH),X86_64)
     HOST_LDOPTS += -framework CoreFoundation
   endif
 endif
 
+
```

Comment:
Nit: Can you remove these two empty lines, please?

Reply:
Fixed.